packages/arb-token-bridge-ui/tests/e2e/specs/withdrawETH.cy.ts

```diff
@@ -1,16 +1,19 @@
 /**
- * When user wants to bridge ETH from L2 to L1
+ * When user wants to bridge native token from L2 to L1
```

Comment:
Similarly, should we rename this file to `withdrawNativeToken.cy.ts` ?

Reply:
Updated